### ISSUE TITLE
GetTypeString() for batch norm inference instance_ptrs give empty string

### ISSUE THREAD

**Issue by bghimireamd** [ROCm/composable_kernel#954]
Currently when I try to print the `GetTypeString()` of a batch norm inference kernel it is giving empty string. 

PS: MIOpen's tuning infrastructure need this unique name.

**Comment by qianfengz** [comment]
`BatchNorm` inference is implemented using `DeviceElementwiseImpl`,  I will add GetTypeString() to `DeviceElementwiseImpl`.  But GetTypeString() is usually used to output tuning information for the Device Instance. For uniquely identifying a Device Instance,  I suggest you to use `GetTypeIdName()` (longer) or `GetTypeIdHashCode()` (shorter)
